core/lib/refinery/core/configuration.rb

```diff
@@ -32,6 +32,7 @@ module Core
     self.visual_editor_javascripts = []
     self.visual_editor_stylesheets = []
     self.plugin_priority = []
+    self.regex_white_list = false
```

Comment:
I think it still use `Loofah` for the configurations `Refinery::Pages::whitelist_elements`and `Refinery::Pages::whitelist_attributes` but use `Rails::Html::PermitScrubber` to create the scrubber intead of `Loofah::Scrubber`